apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/ast/AST.kt

```diff
@@ -0,0 +1,121 @@
+package com.apollographql.apollo.compiler.ast
+
+data class AST(
+  val enums: List<EnumType>,
+  val customTypes: Map<String, String>,
+  val inputTypes: List<InputType>,
+  val fragments: List<FragmentType>,
+  val operations: List<OperationType>
+) {
+  data class OperationType(
```

Comment:
any reason to have this as a nested class? We can have multiple classes in same file in kt

Reply:
There high chances to have name clash if all classes generated in the same package.